.cz-config.js

```diff
@@ -25,7 +25,20 @@ module.exports = {
     { value: 'revert', name: 'revert:   Revert to a commit' },
     { value: 'WIP', name: 'WIP:      Work in progress' },
   ],
-
+  additionalQuestions: [ 
+    {
```

Comment:
Yes, I agree. You could add it to the README docs for this field and leave the other files as is because most people won't use it straight away.

I believe the subject, body, and footer are ideal for most cases and are already decent/solid defaults. 

The new field `additionalQuestions` could exist, no problems with it.

We need to thing about the initial API and keep it as simple as possible for someone new starting to use this package. We don't want them to feel overwhelmed by the number of options.

Reply:
@samarpan-b @leonardoanalista  Yes, got it.
changes done and `additionalQuestions`  removed from .cz-config.js  as its optional field.
we have `additionalQuestions`  in readme and config example file.